spec/factories/custom_metadatum_collection_with_asset_factories.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+FactoryBot.define do
+  factory :custom_metadatum_collection_with_asset,  parent: :custom_metadatum_collection do
```

Comment:
Sorry... stuck record... can we change `asset` terminology to `labware` please? :)

Reply:
This is actually a redundant factory. I'll remove this entire factory.